src/main/webapp/jsp/studentFeedbackSubmissionEdit.jsp

```diff
@@ -1,119 +1,32 @@
-                                            Const.ActionURIs.INSTRUCTOR_EDIT_STUDENT_FEEDBACK_SAVE :
-                                            Const.ActionURIs.STUDENT_FEEDBACK_SUBMISSION_EDIT_SAVE;
-                %>
-                <form method="post" name="form_student_submit_response" action="<%= submitAction %>">
-                    <jsp:include page="<%= Const.ViewURIs.FEEDBACK_SUBMISSION_EDIT %>" />
-
-                    <div class="bold align-center">
-                        <%
-                            if (data.isModeration) {
-                        %>
-                                <input name="moderatedstudent" value="<%= data.studentToViewPageAs.email %>" type="hidden">
-                        <%
-                            }
-
-                            boolean isSubmittable = data.isSessionOpenForSubmission || data.isModeration;
-                            if (data.bundle.questionResponseBundle.isEmpty()) {
-                        %>
-                                There are no questions for you to answer here!
-                        <%
-                            } else if (data.isPreview || !isSubmittable) {
-                        %>
-                                <input disabled="disabled" type="submit" class="btn btn-primary" id="response_submit_button" data-toggle="tooltip" data-placement="top" title="<%= Const.Tooltips.FEEDBACK_SESSION_EDIT_SAVE %>" value="Submit Feedback" style="background: #66727A;">
-                        <%
-                            } else {
-                        %>
-                                <input type="submit" class="btn btn-primary" id="response_submit_button" data-toggle="tooltip" data-placement="top" title="<%= Const.Tooltips.FEEDBACK_SESSION_EDIT_SAVE %>" value="Submit Feedback">
-                        <%
-                            }
-                        %>
-                    </div>
-                    <br><br>
-                </form>
-        </div>
-
-        <jsp:include page="<%= Const.ViewURIs.FOOTER %>" />
-    </body>
-</html>
\ No newline at end of file
+<%@ page import="teammates.common.util.Const"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags/student" prefix="ts" %>
+<%@ taglib tagdir="/WEB-INF/tags/student/feedbackSubmissionEdit" prefix="feedbackSubmissionEdit" %>
+
+<c:set var="jsIncludes">
+    <script type="text/javascript" src="/js/common.js"></script>
+    <script type="text/javascript" src="/js/feedbackSubmissionsEdit.js"></script>
+    <script type="text/javascript" src="/js/student.js"></script>
+</c:set>
+
+<ts:studentPage pageTitle="TEAMMATES - Submit Feedback" bodyTitle="Submit Feedback" jsIncludes="${jsIncludes}">
+    <c:choose>
+        <c:when test="${not data.headerHidden}">
+            <jsp:include page="<%= Const.ViewURIs.STUDENT_HEADER%>" />
+            <jsp:include page="<%= Const.ViewURIs.STUDENT_MOTD%>" />
+        </c:when>
```

Comment:
If you have to deal with `Const.ViewURIs.STUDENT_HEADER` in special ways, you'll most likely have to either edit `studentPage.tag` to add an optional header (navBar) attribute or create a new type of studentPage.